src/Sylius/Behat/Context/Api/Shop/OrderContext.php

```diff
@@ -366,7 +368,7 @@ public function theCustomerServiceShouldKnowAboutThisAdditionalNotes(
 
         Assert::same(
             $notes,
-            $this->responseChecker->getValue($this->adminOrderClient->show($order->getTokenValue()), 'notes')
+            $this->responseChecker->getValue($this->adminClient->show(self::RESOURCE, $order->getTokenValue()), 'notes')
```

Comment:
Nice to be able to spot it, but perhaps we should extract separate context for it? May be done in separate PR